jsx/FilterForm.js

```diff
@@ -162,7 +162,11 @@ class FilterForm extends React.Component {
     }
 
     return (
-      <Panel id="selection-filter" title="Selection Filter">
+      <Panel
+        id={this.props.id}
+        height={this.props.height}
+        title={this.props.title}
```

Comment:
@jacobpenny they are part of Panel.propTypes so, all the warning will be triggered anyways